media/css/sandstone/buttons-new.less

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+input[type="submit"].button-newsletter,
+.button-newsletter {
+
+    background: none;
+    color: #0095dd;
+    border: 2px solid #0095dd;
+    border-radius: 4px;
+    width: 100%;
+    min-height: 45px;
+    .font-size(18px);
+    font-weight: normal;
+    text-shadow: none;
+    box-shadow: none;
+    -webkit-appearance: none;
+    cursor: pointer;
+
+    &:hover,
+    &:focus {
+        color: #00afe5;
```

Comment:
Agree, I was thinking more of naming the variables. Currently we have things like Firefox Red, Mozilla Blue etc. with that said it may, should not?,  be tied to tightly to the identity language as that might change as well. I am open to either though